### ISSUE TITLE
Simple request 02: Option to automatically save changes in permanent sessions

### ISSUE THREAD

**Issue by EstherMoellman** [Emano-Waldeck/Save-Tabs#3]
@Emano-Waldeck , here is my second simple request: Please add an option to automatically save changes in permanent sessions. In simple words, if this option is checked, when the session is closed, every change (new tabs, deleted tabs etc) will be automatically saved under the same name session.

Thank you in advance!

**Comment by RobLockhart** [comment]
I agree, I am used to the likes of a Chrome extension called Session Buddy (which doesn't work in Edge), auto-saves the sessions. Perfect when Edge crashes (and recovery doesn't recover).
